src/Utils.cpp

```diff
@@ -103,13 +106,7 @@ bool utils::IsHeader(const std::string_view& path)
 
 std::string utils::GetNicePath(const std::string_view& path)
 {
-    std::string res(path);
-    ForwardSlashify(res);
-    if (BeginsWith(res, s_CurrentDir))
-        res = res.substr(s_CurrentDir.size(), res.size() - s_CurrentDir.size());
-    if (BeginsWith(res, s_Root))
-        res = res.substr(s_Root.size(), res.size() - s_Root.size());
-    return res;
+    return fs::path(path).lexically_normal();
```

Comment:
This code might "fail" if symlinks are present.
But doing this properly would require lots of file IO I think.